src/constants.ts

```diff
@@ -67,6 +68,17 @@ export const DTYPE_VALUES = {
     type: GL.INT,
     max: 2 ** (32 - 1) - 1,
     sampler: 'isampler2D'
+  },
+  // Cast Float64 as 32 bit float point so it can be rendered.
+  Float64: {
+    format: GL.R32F,
+    dataFormat: GL.RED,
+    type: GL.FLOAT,
+    // Not sure what to do about this one - a good use case for channel stats, I suppose:
+    // https://en.wikipedia.org/wiki/Single-precision_floating-point_format.
+    max: 3.4 * 10 ** 38,
+    sampler: 'sampler2D',
+    cast: (data: TypedArray) => new Float32Array(data)
```

Comment:
Is it not possible to do this without casting?

Reply:
WebGL does not support 64 bit floating point AFAIK.

Reply:
https://github.com/visgl/luma.gl/blob/8731d4c61b69f888c25491a69f0f5d3c96d909f9/modules/constants/src/index.js#L199

Reply:
Just seeing your comment – trust your judgement!

Reply:
I think that is used internally by vis.gl for vertex attributes, but not for textures, but I will ask to be sure: 
https://github.com/visgl/luma.gl/pull/1207
https://github.com/visgl/deck.gl/pull/3468
https://deck.gl/docs/developer-guide/64-bits

Float 64 is not on the Khronos table, hence my original inclination: 
https://webgl2fundamentals.org/webgl/lessons/webgl-data-textures.html

Reply:
In general, it's nice to see how simple it was to accommodate a new datatype without actually touching the `XRLayer`.

Reply:
Confirmed that there is no native 64 bit float texture support, even through deck.gl.

Reply:
We could alter our shaders and split the data into two 32 bit float textures (or some similar hack) and then sample + combine on the fragment shader but this seemed easier.

Reply:
> In general, it's nice to see how simple it was to accommodate a new datatype without actually touching the `XRLayer`. I think this is exactly the right approach. Anyone interacting with the loader gets the "right" data array, and we just cast at the last minute when creating a texture.

Exactly! It's brilliant!